auslib/test/blobs/test_gmp.py

```diff
@@ -49,6 +49,11 @@ def setUp(self):
                     "filesize": 666,
                     "hashValue": "666",
                     "fileUrl": "http://evil.com/fire"
+                },
+                "default": {
+                    "filesize": 20,
+                    "hashValue": "20",
```

Comment:
I'd prefer to have different values for filesize and hashValue, to avoid accidental test passes.